testsuite/domain/src/test/resources/host-configs/host-secrets.xml

```diff
@@ -55,7 +55,7 @@
 
     <domain-controller>
         <!-- Remote domain controller configuration with a host and port -->
-        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm"/>
+        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false"/>
```

Comment:
I don't see any use of this file at all. From the root of the source tree:

$ git grep secrets
testsuite/shared/src/main/java/org/wildfly/test/security/VaultSession.java:            + "keytool -genseckey -alias Vault -storetype jceks -keyalg AES -keysize 128 -storepass secretsecret -keypass secretsecret -keystore %s",